tensorflow/contrib/cmake/tf_c.cmake

```diff
@@ -12,6 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+
+# 1. Resolve the installed version of Python (for Python.h and python).
```

Comment:
Why are these in `tf_c.cmake`? As far as I can tell, there are no files depending on Python.h or numpy/arrayobject.h in this module.

Reply:
auto merge problem. this session was from tf_c.cmake. Removed from latest commit